var/spack/repos/builtin/packages/bolt/package.py

```diff
@@ -19,17 +19,23 @@ class Bolt(CMakePackage):
 
     homepage = "http://www.bolt-omp.org/"
     url      = "https://github.com/pmodels/bolt/releases/download/v1.0b1/bolt-1.0b1.tar.gz"
+    git      = "https://github.com/pmodels/bolt.git"
 
+    version("develop", branch="master")
```

Comment:
You can actually call this version `master` now that #1983 has been merged.
```suggestion
    version("master", branch="master")
```